src/libs/sign.js

```diff
@@ -242,6 +242,19 @@ export const parseXcodebuildError = message => {
   return aLine.substr(key.length).trim();
 };
 
+/**
+ * Parse out the meaningful error message from a failed apksigner sign job
+ *
+ * @param {string} message Multiline message from apksigner build
+ * @returns A `string` containing the meaningful error message
+ */
+export const parseApksignerbuildError = message => {
+  const key = ':';
+  const aLine = message.split(`${key}`)[1].split('\n')[0];
```

Comment:
If the error message changes the index `[1]` or `[0]` might fail and throw an exception. Do you want to catch it here and handle it gracefully?

Reply:
Because you're splitting on just the key you can do `message.split(key)`.

Reply:
right! thanks!